docs/Installation.md

```diff
@@ -51,7 +51,7 @@ $ go mod tidy
 <p>
 
    Some tests could require you have a Github App created in order to test Component builds via Pipelines as Code.
-Such tests are [rhtap-demo](https://github.com/konflux-ci/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go), [build](https://github.com/konflux-ci/e2e-tests/blob/main/tests/build/build.go), and [status-reporting-to-pullrequest](https://github.com/konflux-ci/e2e-tests/blob/main/tests/integration-service/status-reporting-to-pullrequest.go).
+Such tests are [konflux-demo](https://github.com/konflux-ci/e2e-tests/blob/main/tests/konflux-demo/konflux-demo.go), [build](https://github.com/konflux-ci/e2e-tests/blob/main/tests/build/build.go), and [status-reporting-to-pullrequest](https://github.com/konflux-ci/e2e-tests/blob/main/tests/integration-service/status-reporting-to-pullrequest.go).
```

Comment:
```suggestion
Such tests are [rhtap-demo](https://github.com/konflux-ci/e2e-tests/blob/main/tests/konflux-demo/rhtap-demo.go), [build](https://github.com/konflux-ci/e2e-tests/blob/main/tests/build/build.go), and [status-reporting-to-pullrequest](https://github.com/konflux-ci/e2e-tests/blob/main/tests/integration-service/status-reporting-to-pullrequest.go).
```
since we have just renamed the directory but not the test.

Reply:
in fact this is my miss that I didn't the test , 
are you fine if we keep the Installation as is  and I rename the test ?  `tests/konflux-demo/konflux-demo.go`

Reply:
yes, thank you!